### ISSUE TITLE
Unable to select an accidental with keyboard navigation

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#31739]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

There seems to be no way to navigate to an accidental with the keyboard independetly of the note it is attached to in order to change its properties, in particular add a reminder accidental bracket just around the accidental.

STEPS:
1.  Enter a few notes into a score including 1 with an accidental.
2. Use Alt + arrow keys to navigate along element by element.  Notice the navigation moves to the note with the accidental and speech announces this including the accidental, but it skips the accidental itself.
This is correct behaviour for the Alt arrow and we wouldn't want it to move to every accidental independetly of the note it is attached to, but there needs to be some other way to do this, for example Ctrl Alt arrows to make a micro movement to individual aspects  within an element. 

### Supporting files, videos and screenshots

N/A

### In which versions of MuseScore Studio is this issue present?

4.6

### Regression

I was unable to check

### Operating system

Windows 11 with NVDA

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible
